manifests/base/redis/argocd-redis-deployment.yaml

```diff
@@ -21,7 +21,7 @@ spec:
             - admin
             - redis-initial-password
           image: quay.io/argoproj/argocd:latest
-          imagePullPolicy: IfNotPresent
+          imagePullPolicy: Always
```

Comment:
Why do we change this one?

Reply:
Hello, @andrii-korotkov-verkada.

First of all, other places using the `latest` image tag have `imagePullPolicy` set to `Always`.

While working on this issue, I spoke with @pasha-codefresh, @crenshaw-dev, and other contributors and learned why we should use the `Always` option when using the `latest` image tag.

In summary, if we use the `IfNotPresent` option for images with the `latest` tag, newly built images might not be applied when building/testing locally. So, I updated it to `Always` to match the other instances where `quay.io/argoproj/argocd:latest` is used.
(reference: https://github.com/argoproj/argo-cd/issues/20252#issuecomment-2396423786)

Please let me know if you have any other thoughts. :)

Thanks for the comment!